soc/arm/st_stm32/stm32wl/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+# SPDX-License-Identifier: Apache-2.0
+
+zephyr_include_directories(${ZEPHYR_BASE}/drivers)
+zephyr_sources(
```

Comment:
nitpick: line split is not necessary `zephyr_sources(soc.c)`

Reply:
done